lighthouse-core/report/html/report-styles.css

```diff
@@ -739,9 +739,10 @@
 
 .lh-scorescale {
   color: var(--medium-75-gray);
-  padding: 0 var(--section-indent) 0 0;
+  padding: 0 calc(var(--section-indent) * 1.5) 0;
   text-align: right;
   transform-origin: bottom right;
+  will-change: opacity; /* opacity is changed on scroll */
```

Comment:
see the score scales look fine with this now, how are you supposed to tell when the text will be messed up by this?